Application/Features/FilterMovies/FilterMoviesQueryValidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using FluentValidation;
+
+namespace Application.Features.FilterMovies;
+
+public class FilterMoviesQueryValidator : AbstractValidator<FilterMoviesQuery>
+{
+    public FilterMoviesQueryValidator()
+    {
+        When(x => x.MovieFiltersDto is not null, () =>
+        {
+            RuleFor(x => x.MovieFiltersDto!.MaxYear).LessThanOrEqualTo(DateTime.Now.Year)
+                .WithMessage("Year must be less than current year");
+            RuleFor(x => x.MovieFiltersDto!.MaxYear).GreaterThanOrEqualTo(0)
+                .WithMessage("Year must be greater than 0");
+            RuleFor(x => x.MovieFiltersDto!.MinYear).GreaterThanOrEqualTo(0)
```

Comment:
I would change this from 0 to 1800. Is there really some movie from year 457 for example?